dockerfiles/splash/provision.sh

```diff
@@ -65,23 +81,23 @@ install_builddeps () {
 
 install_python_deps () {
     # Install python-level dependencies.
-    pip install -U pip && \
-    /usr/local/bin/pip install --no-cache-dir \
-        Twisted==15.2.1 \
-        qt4reactor==1.6 \
+    pip3 install -U pip && \
+    pip3 install -U https://github.com/twisted/twisted/archive/trunk.zip#egg=twisted && \
```

Comment:
If a latest commit is needed we should pin package versions in Dockerfile, e.g. `https://github.com/twisted/twisted/archive/a5108c0db829847bbfd4bf427868eb0b13da0465.zip`.